Add Prague 2020 website (#1251)
writethedocs/www: docs/conf/prague/2020/news/announcing-virtual-conf-dates.rst

:template: {{year}}/generic.html

.. post:: June 11, 2020

>    :tags: prague-2020, tickets, visiting, sponsors

Reviewer: ```suggestion
   :tags: prague-2020, tickets, sponsors
```

Announcing virtual conference dates
===================================

As we :doc:`announced </blog/moving-portland-prague-2020-online>`, Write the Docs Prague 2020 will be a virtual conference.

> We decided to move the conference to October: the writing day is on

Reviewer: ```suggestion
We decided to move the conference to October, to allow for enough time after the new Portland conference dates in August. The writing day will be on
```

> **{{date.day_two.dotw}}, {{date.day_two.date}}, 9am-5pm {{tz}}**,

Reviewer: ```suggestion
**{{date.day_two.dotw}}, {{date.day_two.date}}, 10:00-17:00 {{tz}}**,
```

> and the main conference days are on **{{date.day_three.dotw}}, {{date.day_three.date}}, 9am-5pm {{tz}}**.

Reviewer: ```suggestion
and the main conference days will be on **{{date.day_three.dotw}}, {{date.day_three.date}}, 10:00-18:00 {{tz}}**.
```

Ticket sales will re-open in the summer. Check out the :doc:`tickets page </conf/{{shortcode}}/{{year}}/tickets>` for prices. Your virtual ticket includes:

> * Live streaming of all talks (available for ticket-holders only)

Reviewer 2: ```suggestion
* Live streaming of all talks
```

Should we remove this, so we have leverage to change it?

Author: I don't know. It's meant to be a contrast from live streaming being freely available to everyone in in-person conferences, so that people know this is a ticket-only benefit.

Reviewer: I think it's better to be explicit and strict about the live-stream availability, because unlike our F2F confs, this is currently the only way to watch the talks. If we decide to change it later, we can reword and announce, but until that happens it's better to spell it out I think.

Thoughts?

Reviewer 3: Should we state that talks will be available as usual after the fact?

Reviewer: For the sake of time management, if we want to get this out today, let's not try to solve all the questions at once please........... <3

Reviewer 2: Seems harder to walk this back if people have bought tickets under the assumption that they will not be livestreamed for free. Easy to add that restriction, but worse to take it away later.

Reviewer: But that's the whole point, IIRC.... Anyway, it looks like we don't spell this out in the Portland tickets page: https://www.writethedocs.org/conf/portland/2020/tickets/

So how about we align with the wording we use for Portland for now?

Reviewer 2: Right, but in the team call we discussed having the livestream be public, so I don't want to sell something when we change it later.

Reviewer: We mentioned it in the team call, but decided against it because of the ticket sales incentive...

Reviewer: Let's figure out the exact restriction once we have Portland going and then we'll see what works. For now I'm going to call it an executive decision and accept Eric's suggestion.

* Q&A with speakers after each talk (may not be available for all speakers)
* Access to the conference chat with all other attendees, speakers and sponsors
* Access to the writing day
* The virtual job fair

If you have chosen to convert your :doc:`previously bought Portland 2020 ticket </conf/portland/2020/ticket-choices>`

> into a virtual pass, you will receive further details by e-mail when we open ticket sales.

Reviewer: ```suggestion
to a virtual pass, you will receive further details by e-mail when we open ticket sales.
```
